doc/api.md

```diff
@@ -219,6 +219,8 @@ to the callback will be an object that conforms to the
 [spf.Event](#spf.event)).  
 `postData: ArrayBuffer | Blob | Document | FormData | null | string | undefined`  
 Optional data to send with the request.  Only used if the method is "POST".  
+`withCredentials: boolean | undefined`
+Optional flag to configure the XHR to send withCredentials or not.
```

Comment:
You should revert this file; it will be auto-generated from src/api.js with the next release and will use the description from the JsDoc.